boilerplates/index.js

```diff
@@ -180,20 +180,35 @@ function initGitRepo(cwd) {
     return
   }
 
-  execSync('git init', { cwd })
-  execSync('git add .', { cwd })
-  execSync(
-    [
-      'git',
-      '-c user.name="Romuald Brillout"',
-      '-c user.email="vite-plugin-ssr@brillout.com"',
-      'commit',
-      '--no-gpg-sign',
-      '--message="boilerplate Vite w/ vite-plugin-ssr"'
-    ].join(' '),
-    { cwd }
-  )
+  let isInitialized = false
+  try {
+    // Attempt to create a git repository.
+    // We hide standard git-related logs from the user, but
+    // do log information for any process that exits with an error.
+    execSync('git init', { cwd, stdio: 'ignore' })
+    isInitialized = true
+
+    execSync('git add .', { cwd, stdio: 'ignore' })
+    execSync(
+      [
+        'git',
+        '-c user.name="Romuald Brillout"',
+        '-c user.email="vite-plugin-ssr@brillout.com"',
+        'commit',
+        '--no-gpg-sign',
+        '--message="boilerplate Vite w/ vite-plugin-ssr"'
+      ].join(' '),
+      { cwd, stdio: 'ignore' }
+    )
+  } catch (error) {
+    if (isInitialized) {
+      fs.rmSync(path.join(cwd, '.git'), { recursive: true, force: true })
```

Comment:
I noticed a note above on L168 about Node 12 support, but it doesn't seem like that is covered any more in test cases? Is it safe to use rmSync now?

Reply:
Yea it's fine.